assayist/processor/utils.py

```diff
@@ -182,7 +182,9 @@ def get_source_of_build(build_info):
     """
     no_source_msg = f'Build {build_info["id"]} has no associated source URL'
     if build_info.get('source'):
-        return build_info['source']
+        source = build_info['source']
+        return source.lstrip('git+') if 'gerrit' in source else source
```

Comment:
Have you consider replacing with regex ?
```
>>> print(test)
git+https://pkgs.com/containers/rsyslog?rhel#4a4109c3e85908b6899b1aa291570f7c7b5a0cb5
>>> re.sub('^git\+http', 'http', test)
'https://pkgs.com/containers/rsyslog?rhel#4a4109c3e85908b6899b1aa291570f7c7b5a0cb5'
```

Reply:
Sure, that will also work. Refactored!

Reply:
It probably works just fine in this scenario, but `lstrip` here won't just remove `git+` if the string starts with it, it will strip off any `g`, `i`, `t`, and `+` at the beginning of the string.

For instance:
```python
>>> 'tigger'.lstrip('git+')
'er'
```